src/ast/expressions.rs

```diff
@@ -22,17 +21,10 @@ pub enum Expression {
     ElementaryTypeNameExpression(ElementaryTypeNameExpression),
     TupleExpression(TupleExpression),
     NewExpression(NewExpression),
-
-    #[serde(rename_all = "camelCase")]
```

Comment:
Remove the UnhandledExpression separately? Probably requires Yul stuff to be added.